common/rfb/ZRLEEncoder.cxx

```diff
@@ -17,6 +17,7 @@
  * USA.
  */
 
+#include <cstdio>
```

Comment:
Sorry, I missed this before. This doesn't look like it is used?

Reply:
You're correct, removed it now.

Reply:
You're correct, removed it.